README.md

```diff
@@ -54,11 +54,11 @@ Your account's Auth Token is hidden by default. Click show to display the token,
 
 ## Quickstart
 
-This sample demonstrates a scenario where the Twilio connector is used to send a text message to a phone number.
+To use the `twilio` connector in your Ballerina application, modify the `.bal` file as follows:
 
 ### Step 1 - Import the package
```

Comment:
```suggestion
### Step 1 - Import the module
```

Reply:
check for other places as well.

Reply:
Changed with [ee499ad](https://github.com/ballerina-platform/module-ballerinax-twilio/pull/229/commits/ee499adf680b027f71751549560724dd5e09a396).